frontend/public/components/modals/remove-volume-modal.tsx

```diff
@@ -72,24 +73,31 @@ export const RemoveVolumeModal: React.FC<RemoveVolumeModalProps> = (props) => {
 
   const { kind, resource, volume } = props;
   const type: string = _.get(getVolumeType(volume.volumeDetail), 'id', '');
+  const { t } = useTranslation();
```

Comment:
Move the call to `useTranslation` to the top of the component function (where `useState` appears). It needs to always be called, and if someone adds an early return someday, it might not be called.